fdbcli/fdbcli.actor.cpp

```diff
@@ -552,6 +552,14 @@ void initHelp() {
 	helpMap["set"] = CommandHelp("set <KEY> <VALUE>",
 	                             "set a value for a given key",
 	                             "If KEY is not already present in the database, it will be created." ESCAPINGKV);
+
+	helpMap["setknob"] = CommandHelp("setknob <KEY> <VALUE> [CONFIG_CLASS]",
```

Comment:
Is this documentation enough? Or we need to update in any other places? For example docs or Dynamic Knobs design doc. cc @sfc-gh-ljoswiak

Reply:
I tried to make it so that existing commands would work and wouldn't need to be changed. The extra argument to begin/commit is optional but I guess it would be nice to add to documentation for the these new knob management commands as well

Reply:
It would be good to add an `fdbcli` section in https://github.com/apple/foundationdb/blob/main/design/dynamic-knobs.md under the `Clients` header.

https://github.com/apple/foundationdb/blob/main/documentation/sphinx/source/command-line-interface.rst should also be updated to reflect the new `setknob` and `getknob` commands, as well as the updated `begin` and `commit` functionality.

Reply:
Should they be added to this PR as well?